### DISCUSSION TITLE
CBT 입점 희망 솔루션 심사 관련 문의

### DISCUSSION THREAD

**Discussion by githubcnsbs** [commerce-api-naver/commerce-solutions#22] [묻고 답하기] [answered]
안녕하세요.

"[커머스솔루션] CBT 입점 희망 솔루션 심사요청일 관련 설문" 내용 중에 이해가 안 가는 부분이 있습니다.

[솔루션 등록 심사 설문]
솔루션 심사 요청은 아래 3가지 조건을 모두 만족한 후 가능합니다.

솔루션 구독 이력: 솔루션을 구독한 계정이 존재한 이력이 있어야 심사 요청이 가능합니다.
솔루션 IP 등록: API호출을 위한 IP가 1개 이상 등록되어야 심사 요청이 가능합니다.
솔루션 등록하기 > 솔루션 애플리케이션 > API호출 IP를 등록 부탁드립니다.
API 그룹 추가 사유 입력
솔루션 등록하기 > 솔루션 애플리케이션 > API사용 > 선택한 API의 'API 그룹 추가 사유'를 200자 이상 작성 필요합니다.
위 내용 중 솔루션 구독 이력이란 어떤 것일까요.
솔루션 등록 전에 타 솔루션 구독 이력이 있어야 심사 요청이 가능하다는 것일까요.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

'솔루션 구독 이력: 솔루션을 구독한 계정이 존재한 이력이 있어야 심사 요청이 가능합니다.' 항목의 경우 아래와 같은 배경으로 포함된 조건입니다.
- 모든 솔루션은 사용자 분들께 정식으로 공개(서비스 오픈)하기 이전 개발사 내부에서 충분한 연동 테스트를 진행해봐야 합니다.
- 개발사에서는 연동중인 솔루션에 대해 개발사 명의의 테스트용 계정(커머스ID 계정)으로 연동중인 솔루션을 커머스솔루션마켓에서 직접 구독하여 사용자 입장에서 연동이 원활하게 진행되는지 테스트를 해보아야 합니다.

따라서 해당 내용은 타 솔루션 구독 이력이 아닌 연동중인 자사 솔루션에 대한 테스트성 구독 이력을 요구하는 내용으로 이해해주시기 바랍니다.

**Reply by githubcnsbs** [reply]
네, 답변 감사합니다. 

그런데 솔루션 등록이 안된 상태에서 솔루션 구독 테스트가 가능한가요. 
방안 가이드 위치가 있으면 공유 부탁드립니다.

**Reply by commerce-api-naver** [reply]
@githubcnsbs 님, 커머스API 이민석입니다.

해당 설문은 CBT 프로그램에 참가하신 솔루션 개발사를 대상으로 솔루션 연동을 어느정도 마무리하여 커머스솔루션마켓에 공개하기 전 심사 일정을 묻는 용도로 발송된 설문입니다.

솔루션 등록이 안 된 상태라면 솔루션 테스트조차 못하신 것으로 추정할 수 밖에 없는데요,
혹시 개발사 이름이나 솔루션명을 알 수 있을까요?

**Reply by githubcnsbs** [reply]
심사 절차를 잘못 이해한 것 같습니다. 
심사 절차에 대한 가이드 wiki가 있을까요

**Reply by commerce-api-naver** [reply, marked as answer]
@githubcnsbs 님 안녕하세요. 커머스API 이민석입니다.

현재 심사 절차와 관련한 별도 가이드는 제공하지 않고 있습니다.
조만간 커머스API센터 내 솔루션 심사/관리 기능을 정식으로 제공할 예정으로 이 과정에서 관련된 가이드 역시 함께 제공할 예정입니다.
